### PR TITLE
Add Storefront API link to README

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ TBD: Add Redocly validation status badge.
 - [Core API docs](https://api-reference.rebilly.com/)
 - [Users API docs](https://user-api-docs.rebilly.com/)
 - [Reports API docs](https://reports-api-docs.rebilly.com/)
+- [Storefront API docs](https://storefront-api-docs.rebilly.com/)
```
Comment (ashkarpetin) [inline README.md:10]: It requires authentication

Comment (slavcodev) [reply README.md:10]: Right. What is your concern?

Comment (ashkarpetin) [reply README.md:10]: It's a public repo, all links should be public

Comment (slavcodev) [reply README.md:10]: Why all links should be public? As you said the repository is public, the API defnitions is open-sourced, so existence of such API is not a big secret. I have no idea why the docs is closed for auth users only, probably to limit integrations, but I don't see a reason to not have a links for those who have an access to it.

**Review by ashkarpetin — APPROVED** [review]

**Review by VladimirBerdnik — APPROVED** [review]

**Pull request merged**